code/game/objects/structures/HMG.dm

```diff
@@ -18,7 +18,7 @@
 	var/spread = 10
 	var/damage_modifier = 25
 	var/penetration_modifier = 10
```

Comment:
Lower this by a good chunk for testing.

Reply:
this is the default stat, you need it changed? @Malerus